scripts/ci/ansible/inventory.yml

```diff
@@ -0,0 +1,11 @@
+localhost
```

Comment:
need to remove this file :)

Reply:
I forgot to ask @lebauce if it was a tricks

Reply:
the inventory does have to be submitted, only used during the deployment